lib/core/services/api/api_service_impl.dart

```diff
@@ -163,4 +163,15 @@ class ApiServiceImpl extends ApiService {
       rethrow;
     }
   }
+
+  @override
+  Future<void> addShoeItem(Shoe addShoe) async {
+    print(addShoe.toJSON());
```

Comment:
Rename addShoeItem method parameter addShoe to shoeItem
`Future <void> addShoeItem(Shoe shoeItem);`